label_weeds.py

```diff
@@ -124,6 +125,26 @@ def add_images(self):
                     if not color_path in self.data:
                         self.data[color_path] = {"nir": nir_path}
 
+    def update_directory(self):
```

Comment:
Nechci zdržovat, ale nechceš to raději dát do samostatného skriptu? Tady mi to nepřijde systémový. Nepatří to sem.